var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -16,15 +16,18 @@ class SuperluDist(CMakePackage, CudaPackage):
 
     tags = ['e4s']
 
-    maintainers = ['xiaoye', 'gchavez2', 'balay', 'pghysels']
+    maintainers = ['xiaoye', 'gchavez2', 'balay', 'pghysels', 'liuyangzhuan']
```

Comment:
@xiaoye @gchavez2 @balay @pghysels can you review this change?

Reply:
I mean, I'm a member of the superlu-dist team. But to accelerate this PR, I can remove myself from the maintainers.

Reply:
Sorry, by "this change" I meant the other changes to this package, not specifically that you added yourself to the list of maintainers. I would _love_ to see more maintainers. Thanks for volunteering to maintain so many packages!

Reply:
Sorry I misunderstood that. Of course! I'm happy to hear feedbacks.

Reply:
@adamjstewart I've got some feedback from @pghysels and made some changes in https://github.com/spack/spack/pull/26936/commits/878e6ea16bcf7cc46ca23e045831bf9ffa074199
Can you see if it looks good now?